### ISSUE TITLE
Chrome sign in dialog is totally inaccessible.

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#5838]
I don't know who to report this to, but, in both chrome canary and stable, with NVDA Next, if you go to settings>sign iin, and then try to sign in, it fails. NVDA reports an embeded object and that's all besides sometimes one unlabeled button.

**Comment by jcsteh** [comment]
Duplicate of #5512 (which is just tracking [Chromium issue 589325](https://bugs.chromium.org/p/chromium/issues/detail?id=589325)).
